start.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import re
+from os.path import expanduser
+
+from clusterdock.models import Cluster, Node
+from clusterdock.utils import wait_for_condition
+
+logger = logging.getLogger('clusterdock.{}'.format(__name__))
+
+DEFAULT_NAMESPACE = 'clusterdock'
+DEFAULT_OPERATING_SYSTEM = 'centos6.6'
+
+KDC_ACL_FILENAME = '/var/kerberos/krb5kdc/kadm5.acl'
+KDC_CONF_FILENAME = '/var/kerberos/krb5kdc/kdc.conf'
+KERBEROS_VOLUME_DIR = '/etc/clusterdock/kerberos'
```

Comment:
Nit: move this to end of list to keep all the KDC options together

Reply:
Done